code/workspaces/web-app/src/components/notebooks/NotebookSharingOptions.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  select: {
+    whiteSpace: 'normal',
+    maxWidth: 500,
+  },
+  warning: {
+    color: 'hsl(0, 40%, 40%)',
+  },
+}));
+
+const Personal = () => {
+  const classes = useStyles();
+  return (
+        <div className={classes.select}>
+            <Typography variant="body1">Personal</Typography>
+            <Typography variant="body2">
+                Only you can can view and interact with this notebook.
+            </Typography>
+        </div>
+  );
+};
+
+const Project = () => {
+  const classes = useStyles();
+  return (<div className={classes.select}>
+        <Typography variant="body1">Project</Typography>
+        <Typography variant="body2">Anyone in this project can view and interact with this notebook.</Typography>
```

Comment:
I think these should use `body1` rather than `body2` as `body2` is smaller and was styled to be used for the footer. This might also help with the spacing between elements mentioned above.